src/adaXT/base_model.pyx

```diff
@@ -62,19 +65,19 @@ class BaseModel:
                     self.leaf_builder_class = leaf_builder
                 else:
                     self.leaf_builder_class = LeafBuilderRegression
-            elif tree_type == "LinearRegression":
+            elif tree_type == "Gradient":
                 if predict:
                     self.predict_class = predict
                 else:
-                    self.predict_class = PredictLinearRegression
+                    self.predict_class = PredictLocalPolynomial
                 if criteria:
                     self.criteria_class = criteria
                 else:
-                    self.criteria_class = Linear_regression
+                    self.criteria_class = Partial_quadratic
                 if leaf_builder:
                     self.leaf_builder_class = leaf_builder
                 else:
-                    self.leaf_builder_class = LeafBuilderLinearRegression
+                    self.leaf_builder_class = LeafBuilderPartialQuadratic
 
         else:
```

Comment:
Quantile is missing a check setting its defaults.

Reply:
Should also be set correctly?